packages/data-stores/src/user/types.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+
+import { Action } from 'redux';
+
+export const enum ActionType {
+	CREATE_ACCOUNT = 'CREATE_ACCOUNT',
+	RECEIVE_CURRENT_USER = 'RECEIVE_CURRENT_USER',
+	RECEIVE_CURRENT_USER_FAILED = 'RECEIVE_CURRENT_USER_FAILED',
+	FETCH_CURRENT_USER = 'FETCH_CURRENT_USER',
+	RECEIVE_NEW_USER = 'RECEIVE_NEW_USER',
+	RECEIVE_NEW_USER_SUCCESS = 'RECEIVE_NEW_USER',
```

Comment:
Is the constant name mismatch intentional? Just saw this.

I can't see where `RECEIVE_NEW_USER_SUCCESS` is used anywhere.

Reply:
I'll make a note to remove this, it was there when I thought we'd need to specifically handle a success action separately from dealing with the response data, but didn't end up needing it.